### PR TITLE
Adding translations for congratulations.md

### PR DESCRIPTION
**Pull request by rachnakumar**
Adding es, id, ko, ja, pt_BR, zh_CN translations for congratulations.md (visual stories)

### CODE REVIEW

File: content/docs/getting_started/visual_story/congratulations@es.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+$title: ¡Bien hecho!
+$order: 8
+---
```
[the remark this answers is not in the document]
Comment (rachnakumar) [reply content/docs/getting_started/visual_story/congratulations@es.md]: Updated all the translation files with the front matter changes.

**Comment by CrystalFaith** [comment]
Can I get a ko language review from @techhtml or @captain-lee ? Thanks!

File: content/docs/getting_started/visual_story/congratulations@ko.md [hunk 2]
```diff
@@ -2,24 +2,25 @@
...
 
-- [amp-story 레퍼런스 문서](/ko/docs/reference/components/amp-story.html)
-- [AMP 스토리 베스트 프랙티스](/ko/docs/fundamentals/amp_story_best_practices.html)
+- [amp-story 참조 문서](/ko/docs/reference/components/amp-story.html)
+- [AMP 스토리 권장사항](/ko/docs/fundamentals/amp_story_best_practices.html)
 
-{% call callout('노트', type='note') %}
-[amp-story](/ko/docs/reference/components/amp-story.html) 컴포넌트는 현재 **실험 상태**입니다.
+{% call callout('참고', type='note') %}
+[amp-story](/ko/docs/reference/components/amp-story.html) 구성요소는 아직 **실험 단계**에 있습니다. 이 구성요소를 사용하여 페이지를 게시하기 위한 Origin 평가판에 가입하려면 <a href="http://bit.ly/amp-story-signup">bit.ly/amp-story-signup</a>을 방문하세요.
```
Comment (captain-lee) [inline content/docs/getting_started/visual_story/congratulations@ko.md]: "Origin 평가판에" -> "Origin Trial에": It loses the meaning when it's half translated.

**Comment by pbakaus** [comment]
Confused as this says approved, does this still need to be updated? @CrystalFaith

File: content/docs/getting_started/visual_story/congratulations@ko.md (hunk 2 above)
Comment (captain-lee) [reply content/docs/getting_started/visual_story/congratulations@ko.md]: Looks good to me.

**Comment by captain-lee** [comment]
Looks good to me. No more update is needed for KO.

**Comment by CrystalFaith** [comment]
@pbakaus Sorry PB, this one is ready to go!

**Pull request merged**